batchflow/models/torch/base.py

```diff
@@ -1669,7 +1669,7 @@ def convert_outputs(self, outputs):
 
 
     # Store model
-    def save(self, path, use_onnx=False, path_onnx=None, use_openvino=False, path_openvino=None,
+    def save(self, path, format="pt", pickle_metadata=False,
```

Comment:
```suggestion
    def save(self, path, fmt="pt", pickle_metadata=True,
```

Reply:
`format` is a name of a standard function in Python, try to avoid such names for variables if it is possible (like `slice`, `max`, etc)